### PR TITLE
internal: Change trace too large message to debug

### PR DESCRIPTION
**Pull request by ZStriker19**
Essentially the issue is that we don't accept logs bigger than 256kb: "Any log exceeding 256KB is accepted and truncated by server" and when we log that a trace payload is too large, we print the whole trace, causing a log larger than the acceptable size: https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/api.py#L229

This causes many error level logs to be generated

Which is causing the customer's monitor for error level logs on this application to alert.

### CODE REVIEW

File: ddtrace/api.py [hunk 1]
```diff
@@ -226,7 +226,7 @@ def send_traces(self, traces):
                             payload.add_trace(trace)
                         except PayloadFull:
                             # If the trace does not fit in a payload on its own, that's bad. Drop it.
-                            log.warning('Trace %r is too big to fit in a payload, dropping it', trace)
+                            log.debug('Trace %r is too big to fit in a payload, dropping it', trace)
```
Comment (brettlangdon) [inline ddtrace/api.py:229]: We should keep this as a warning, but maybe we can format the message differently so it isn't as long.

Comment (brettlangdon) [reply ddtrace/api.py:229]: re: keeping as a warning. we really want to make sure these aren't hidden, we want people to know if a trace is every dropped so they can take actions. making a `debug` will effectively hide this from the customers

**Comment by ZStriker19** [comment]
Let me know if you'd rather have something like the trace id of the parent span or something printed when this error occurs. I was not sure how to add that instead from the Trace object.

**Comment by jd** [comment]
I think that's good enough. That should not happen anyhow, and I don't think having any id will help. 🤷  Thank you @ZStriker19!

**Pull request merged**